lib/hooks/updateEmailPreference.ts

```diff
@@ -2,7 +2,7 @@ import { supabase } from "lib/utils/supabase";
 
 interface UpdateEmailProps {
   display_email?: boolean;
-  receive_collaboration?: boolean;
+  receive_connections?: boolean;
```

Comment:
This is going to fail when it hits the API

Reply:
missed this 🥹